rules/rules_test.go

```diff
@@ -563,23 +563,23 @@ func TestExpressions(t *testing.T) {
 		{
 			expr: `round(-1 * (1 + 0.005 * http_requests{group="production",job="api-server"}))`,
 			output: []string{
-				`{group="production", instance="0", job="api-server"} => -2 @[%v]`,
+				`{group="production", instance="0", job="api-server"} => -1 @[%v]`,
```

Comment:
I'd expect some people to be surprised no matter what behaviour we chose for negative numbers, and are unlikely to be able to please everyone. Floating point inaccuracy will also cause some surprises. Overall I went for simplicity of implementation.